docs/asciidoc/servers.adoc

```diff
@@ -329,3 +333,58 @@ none of which are supported, an exception will be thrown.
 - TLS 1.3 support in OpenJDK is (beside Azul's OpenJSSE) expected to come into 8u272.
 - Java 11.0.3 or higher.
 ====
+
+=== HTTP/2 Support
+
+HTTP2 support is provided across web server implementation. To enable it, you must add one of the
+following dependencies:
+
+HTTP/2 with Jetty:
+[dependency, artifactId="jooby-http2-jetty"]
+.
+
+HTTP/2 with Netty:
+[dependency, artifactId="jooby-http2-netty"]
+.
+
+HTTP/2 with Undertow:
+[dependency, artifactId="jooby-http2-undertow"]
+.
+
+Once the required dependencies is add it, Jooby automatically configures HTTP/2.
```

Comment:
Typo: Once the required dependencies *are added*, Jooby automatically configures HTTP/2.